src/app/ngrx/effects/castle.effects.ts

```diff
@@ -144,6 +146,7 @@ export class CastleEffects {
   );
 
   // on episode page success if loading all episode pages and not yet finished, load next page
+  /* Episode loading TBD but this isn't needed for now
```

Comment:
We'll talk more about episode loading and selection soon, but for now stop loading episodes page by page and only load episode pages when selected in pager.